src/Test.sol

```diff
@@ -225,6 +225,62 @@ abstract contract Test is DSTest, Script {
         assertEq0(a, b, err);
     }
 
+    function assertEq(uint256[] memory a, uint256[] memory b) internal {
```

Comment:
This PRis already merged, but curious what you think of just verifying array equality like this instead? Emitting the full arrays on failure seems like a nice way to report mismatched arrays (at least it was helpful for me when writing that lib), and the code is simpler: https://github.com/mds1/solidity-generators/blob/dcb3eeeabbc8c6fb6684617fe2344a8442f8f17e/test/Generators.t.sol#L27-L34

Reply:
Won't this approach also result in duplicate logs? First you'll get `emit log("Error: a[] == b[] not satisfied");` emitted, then you'll also get `emit log("Error: a == b not satisfied [address]");` emitted

Reply:
Yeah, I was aware of how the logs worked.

Your solution is cleaner and we should indeed go with the expected/actual approach. I overlooked it.

That being said, it is only a matter of refactoring the code and changing the logs, no breaking changes. Do you want to reimplement it @mds1?

Reply:
I also agree that @mds1's solution is more elegant. I didn't even think about using `keccak256`!